projects/iio_demo/src.mk

```diff
@@ -62,31 +61,37 @@ endif
 
 # stm32
 ifeq (stm32, $(PLATFORM))
-SRCS += $(STM32CUBE)/STM32CubeF4/Drivers/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal.c \
-	$(STM32CUBE)/STM32CubeF4/Drivers/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal_cortex.c \
-	$(STM32CUBE)/STM32CubeF4/Drivers/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal_rcc.c \
-	$(STM32CUBE)/STM32CubeF4/Drivers/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal_uart.c \
-	$(STM32CUBE)/STM32CubeF4/Drivers/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal_gpio.c
-
 SRCS += $(PLATFORM_DRIVERS)/stm32_delay.c \
 	$(PLATFORM_DRIVERS)/stm32_uart.c \
-	$(PLATFORM_DRIVERS)/stm32_uart_stdio.c \
-	$(PROJECT)/src/hal/stm32/stm32f4xx_hal_msp.c \
-	$(PROJECT)/src/hal/stm32/stm32f4xx_it.c \
-	$(PROJECT)/src/hal/stm32/sysmem.c \
-	$(PROJECT)/src/hal/stm32/system_stm32f4xx.c
-
-ASM_SRCS = $(PROJECT)/src/startup/startup_stm32f446retx.s
-
-INCS += $(PROJECT)/src/hal/stm32/main.h \
-	$(INCLUDE)/delay.h \
-	$(PLATFORM_DRIVERS)/stm32_uart_stdio.h \
-	$(PROJECT)/src/hal/stm32/stm32f4xx_hal_conf.h \
-	$(PROJECT)/src/hal/stm32/stm32f4xx_it.h \
-	$(PROJECT)/src/hal/stm32/stm32_hal.h \
-	$(PLATFORM_DRIVERS)/stm32_uart.h
-
-LSCRIPT = $(PROJECT)/src/linker/STM32F446RETX_FLASH.ld
+	$(PLATFORM_DRIVERS)/stm32_uart_stdio.c
+INCS += $(PLATFORM_DRIVERS)/stm32_uart_stdio.h \
+	$(PLATFORM_DRIVERS)/stm32_uart.h \
+	$(PLATFORM_DRIVERS)/stm32_hal.h
+# SRCS += $(STM32CUBE)/STM32CubeF4/Drivers/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal.c \
```

Comment:
Is there a reason to keep them as comment?

Reply:
i forgot the comments, removed in v4